### PR TITLE
Improve the example of declaring the command name

### PR DESCRIPTION
**Pull request by serhiyzhovnir**
## Purpose of this pull request

This pull request (PR) improves an example of declaring the command name.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.2/extension-dev-guide/cli-cmds/cli-howto.html
-  https://devdocs.magento.com/guides/v2.3/extension-dev-guide/cli-cmds/cli-howto.html

## Links to Magento source code

The Magento uses the **symfony/console** package to create a console command.
The Magento gives us an ability to use the DI arguments (https://devdocs.magento.com/guides/v2.3/extension-dev-guide/build/di-xml-file.html#argument-types) to configure arguments for a class's constructor.
In case if we create a new command and inherits the https://github.com/symfony/console/blob/master/Command/Command.php class, we are able to configure the $name argument for a new command via ID argument.
It makes the command name extendable from another module.
There is no example in the Magento of using this way, but I've tested it and it works well.

-  https://github.com/symfony/console/blob/master/Command/Command.php#L74

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by jeff-matthews** [comment]
@serhiyzhovnir,

There are a few linting errors:

```
rake test:md
Testing Markdown style with mdl ...
src/guides/v2.2/extension-dev-guide/cli-cmds/cli-howto.md:72: MD009 Trailing spaces
src/guides/v2.2/extension-dev-guide/cli-cmds/cli-howto.md:94: MD029 Ordered list item prefix
src/guides/v2.2/extension-dev-guide/cli-cmds/cli-howto.md:116: MD029 Ordered list item prefix
src/guides/v2.3/extension-dev-guide/cli-cmds/cli-howto.md:72: MD009 Trailing spaces
src/guides/v2.3/extension-dev-guide/cli-cmds/cli-howto.md:94: MD029 Ordered list item prefix
src/guides/v2.3/extension-dev-guide/cli-cmds/cli-howto.md:116: MD029 Ordered list item prefix

A detailed description of the rules is available at https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md

The Markdown linter detected 6 issue(s)
```

File: src/guides/v2.2/extension-dev-guide/cli-cmds/cli-howto.md [hunk 1]
```diff
@@ -91,11 +92,17 @@ Following is a summary of the process:
    {:.bs-callout-info}
    Style the output text by using `<error>`, `<info>`, or `<comment>` tags. See [Symfony](https://symfony.com/doc/master/console/coloring.html){:target="_blank"} docummentation for more information about styling.
 
-1. Declare your Command class in `Magento\Framework\Console\CommandListInterface` using dependency injection (`<your component root dir>/etc/di.xml`):
+2. Declare your Command class in `Magento\Framework\Console\CommandListInterface` and configure the command name using dependency injection (`<your component root dir>/etc/di.xml`):
```
Comment (jeff-matthews) [inline src/guides/v2.2/extension-dev-guide/cli-cmds/cli-howto.md]:
```suggestion
1. Declare your Command class in `Magento\Framework\Console\CommandListInterface` and configure the command name using dependency injection (`<your component root dir>/etc/di.xml`):
```

File: src/guides/v2.2/extension-dev-guide/cli-cmds/cli-howto.md [hunk 2]
```diff
@@ -107,7 +114,7 @@ Following is a summary of the process:
    </config>
    ```
 
-1. Clean the [cache](https://glossary.magento.com/cache) and compiled code directories:
+3. Clean the [cache](https://glossary.magento.com/cache) and compiled code directories:
```
Comment (jeff-matthews) [inline src/guides/v2.2/extension-dev-guide/cli-cmds/cli-howto.md]:
```suggestion
1. Clean the [cache](https://glossary.magento.com/cache) and compiled code directories:
```

File: src/guides/v2.2/extension-dev-guide/cli-cmds/cli-howto.md [hunk 3]
```diff
@@ -62,14 +62,15 @@ Following is a summary of the process:
...
-               $this->setName('my:first:command');
                $this->setDescription('This is my first console command.');
                $this->setDefinition($options);
 
                parent::configure();
            }
 
            /**
+            * Execute the command
+            *
```
Comment (jeff-matthews) [inline src/guides/v2.2/extension-dev-guide/cli-cmds/cli-howto.md]:
```suggestion
            *
```

**Comment by serhiyzhovnir** [comment]
Hi @jeff-matthews 
I've added the suggested changes.
Thank you!

**Pull request merged**